src/core/Search/styles.ts

```diff
@@ -0,0 +1,13 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
```

Comment:
While typing the border color should be `highlight`

Reply:
Is it ok like this?


![Screenshot from 2020-10-27 13-02-45](https://user-images.githubusercontent.com/45940691/97279710-c5d01500-1854-11eb-9456-791deeda887c.png)
![Screenshot from 2020-10-27 13-02-52](https://user-images.githubusercontent.com/45940691/97279713-c668ab80-1854-11eb-8b8c-510d1f872f0a.png)
![Screenshot from 2020-10-27 13-02-57](https://user-images.githubusercontent.com/45940691/97279714-c799d880-1854-11eb-9002-6ba5a438c81c.png)

Reply:
Yes this looks good 👍🏻